// Â© OpenShot Studios, LLC
//
// SPDX-License-Identifier: LGPL-3.0-or-later

// [START declaration]
syntax = "proto3";
package pb_tracker;

import "google/protobuf/timestamp.proto";
// [END declaration]

// [START messages]
message Frame {
  int32 id = 1;  // Frame ID.
  float rotation = 2;

  message Box {
    float x1 = 1;
    float y1 = 2;
    float x2 = 3;
    float y2 = 4;
  }

  Box bounding_box = 3;
}

message Tracker {
  repeated Frame frame = 1;

  google.protobuf.Timestamp last_updated = 2;
}
// [END messages]
